Shared.Tests/CloudinaryTest.cs

```diff
@@ -1297,6 +1301,54 @@ public void TestResourcesListingDirection()
             }
         }
 
+        [Test]
+        public void TestListResourcesByContextMetadata_Exception_If_Key_Not_Specified()
```

Comment:
Inconsistent naming convention. Looks not bad. Maybe we can switch to this syntax after some major refactoring. For now make it consistent with other methods